### DISCUSSION TITLE
Fancybox Gallery - How do I link gallery images to external urls?

### DISCUSSION THREAD

**Discussion by visaltgl** [fancyapps/ui#44] [Q&A]
Can I link gallery images to external URLs by using the 'click {String|null}' in Fancybox? Or using the caption area is the only way?

**Comment by fancyapps** [comment]
Hi,

What exactly is your goal? If you want to have a download button, the next version (to be released this week) will provide a toolbar with a download button (with a slideshow, etc.). Also, it will be easier to customize caption to include external links, if needed.

**Reply by fancyapps** [reply]
FYI, The first beta was released yesterday and you can now choose to display download button in the toolbar,  `caption` option makes it easier to customize captions, see these demos:

https://gist.lipsum.app/gists/wa 
https://gist.lipsum.app/gists/w9
